app/controllers/concerns/learning_completion.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module LearningCompletion
+  extend ActiveSupport::Concern
+
+  private
+
+  def practice_tweet_url
```

Comment:
`practice_tweet_url`なのであれば`Practice#tweet_url`でいいかも？

Reply:
> practice_tweet_urlなのであればPractice#tweet_urlでいいかも？

どうして`Practice#tweet_url` の方がいいのでしょうか？
自分でも考えてみたのですが、わかりませんでした🙇‍♂️

また`Practice#tweet_url` だとrubocopが通りません💦
```
app/controllers/concerns/learning_completion.rb:8:1: C: [Corrected] Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).
# tweet_url
^^^^^^^^^^^
app/controllers/concerns/learning_completion.rb:8:15: C: [Corrected] Layout/LeadingCommentSpace: Missing space after #.
  def Practice#tweet_url
              ^^^^^^^^^^
app/controllers/concerns/learning_completion.rb:8:15: C: [Corrected] Layout/SpaceBeforeComment: Put a space before an end-of-line comment.
  def Practice#tweet_url
              ^^^^^^^^^^
app/controllers/concerns/learning_completion.rb:8:15: C: [Corrected] Style/CommentedKeyword: Do not place comments on the same line as the def keyword.
  def Practice#tweet_url
              ^^^^^^^^^^
app/controllers/concerns/learning_completion.rb:8:16: C: [Corrected] Style/CommentedKeyword: Do not place comments on the same line as the def keyword.
  def Practice # tweet_url
               ^^^^^^^^^^^
app/controllers/concerns/learning_completion.rb:9:7: C: Naming/MethodName: Use snake_case for method names.
  def Practice
      ^^^^^^^^
```
rubocopが通るように修正したら以下のようになるのですが、この書き方でも大丈夫でしょうか？
```
# tweet_url
def practice
```

Reply:
@R-Tsukada 
修正を巻き取っていただきありがとうございます🙏
誤解が生じてそうなので補足すると、 `Practice#tweet_url` は 「 `Practice` クラスの `tweet_url` というインスタンスメソッド」を意味します
https://docs.ruby-lang.org/ja/latest/doc/symref.html#num

> Range#each
説明文の中でのみ使われます。Ruby言語の要素ではありません。クラスのインスタンスメソッドであることを簡単に表示するための表記法です。一方、クラスメソッドは「Range.new」のように「.」でつなぎます。

Reply:
@konaga-k 
お返事が遅くなってしまって申し訳ありません🙇‍♂️

> 誤解が生じてそうなので補足すると、 Practice#tweet_url は 「 Practice クラスの tweet_url というインスタンスメソッド」を意味します

とても誤解していました💦ありがとうございます🙇‍♂️

遅くなってしまいましたが、修正しました！